### ISSUE TITLE
Add maplin to dissolved list

### ISSUE THREAD

**Issue by doublah** [osmlab/name-suggestion-index#3424]
https://en.wikipedia.org/wiki/Maplin_(retailer) says brand closed all stores on 25 June 2018.

**Comment by hikemaniac** [comment]
I took care of adding the information to Wikidata and I also added the inception date of the online store (though I'm not entirely sure to have done it sensibly). Could someone with more experience please check it on [https://www.wikidata.org/wiki/Q6754124](https://www.wikidata.org/wiki/Q6754124)?
Anyway, the next time `npm run wikidata` is run it should be added to dissolved.json. 👍

**Comment by boothym** [comment]
@bhousel should Maplin still be in the NSI given it's closed? https://nsi.guide/index.html?k=shop&v=electronics#Maplin

**Comment by bhousel** [comment]
Yes @boothym - we still include the closed brands in the index. They don't actually get included in the dist/ preset files that browsers pick up, but it might be useful for people to work with them, for example to run an overpass query and close them in OSM.

Issue #3800 is about showing this info on the https://nsi.guide site (which we don't do yet).

**Comment by boothym** [comment]
Thanks. Would definitely be useful to have "Closed" or something else in the Maplin row to indicate it's closed.

It's still available as a preset in iD though - I assume that is because iD hasn't picked up the updated files recently, rather than an NSI issue?
